recipes/latentstrainanalysis/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "LatentStrainAnalysis" %}
+{% set version = "1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: 4db66e9
+  git_url: https://github.com/douglasgscofield/{{ name }}.git
```

Comment:
Any special reason you are using Douglas' fork? I think it would be better to select the original repo. In that case you can use `https://github.com/brian-cleary/LatentStrainAnalysis/archive/7f1846849a4fe374897de70b55bd65662ba18912.ta.gz` as download link. 

You can always build the download link like that: gitrepo/projectname/archive/commit_id.tar.gz